packages/field-plugin/src/createFieldPlugin/FieldPluginData.ts

```diff
@@ -3,9 +3,9 @@
  */
 import { StoryData } from '../messaging'
 
-export type FieldPluginData = {
+export type FieldPluginData<Content = unknown> = {
```

Comment:
Using `unknown` as a fallback, in case someone want to use this type and is uninterested in the `content`.